build.sbt

```diff
@@ -218,7 +218,7 @@ lazy val docs = project       // new documentation project
     scalacOptions += "-language:reflectiveCalls",
     mdocIn := file("docs/src"),
     mdocOut := file("docs/generated"),
-    mdocExtraArguments := Seq("--cwd", "docs"),
+    mdocExtraArguments := Seq("--cwd", "docs", "--no-link-hygiene" ),
```

Comment:
Is this something that we want? Or: do we have dead links? (Is that what this is checking?)

Reply:
All the links when viewed as mdoc (vs website) are broken because markdown/GitHub wants them to be called "foo.md" but the website wants the link to be "foo". I can revert this change, it was just noisy while I was iterating

Reply:
All of our links look broken to mdoc but work on the website. I think there's got to be a better solution but at the moment it's noisy for no reason 🤷

Reply:
I'm going revert this change because it's still helpful if you want to manually look at the list and see there is nothing that isn't of the "did you mean foo.md" type